x-pack/plugins/reporting/server/export_types/csv_searchsource/generate_csv/generate_csv.ts

```diff
@@ -84,7 +84,13 @@ export class CsvGenerator {
     try {
       results = (
         await lastValueFrom(
-          this.clients.data.search(searchParams, { strategy: ES_SEARCH_STRATEGY })
+          this.clients.data.search(searchParams, {
+            strategy: ES_SEARCH_STRATEGY,
+            transport: {
+              maxRetries: 1, // retrying reporting jobs is handled in the task manager scheduling logic
+              requestTimeout: this.config.get('csv', 'scroll', 'duration'),
```

Comment:
I'm not sure if this string is validated to be a legit timeout value. Needs investigation.